infrastructure/network/connmanager/outgoing_connections.go

```diff
@@ -24,46 +28,26 @@ func (c *ConnectionManager) checkOutgoingConnections(connSet connectionSet) {
 
 	connectionsNeededCount := c.targetOutgoing - len(c.activeOutgoing)
 	connectionAttempts := connectionsNeededCount * 2
```

Comment:
We no longer need the `* 2` here.
We needed it before, because much of the connectionsAttempts were discovered as existing/banned.
Since RandomAddresses no longer returns existing/banned addresses, we should be fine.